### PR TITLE
added the projects , ideas and thoughts controllers

### PR DESCRIPTION
**Pull request by khushilms**

### CODE REVIEW

File: src/routes/v1/idea.route.js [hunk 1]
```diff
@@ -2,14 +2,14 @@ const express = require('express');
...
 
 const router = express.Router();
 
 router
-  .route('/')
-  // .post(auth('manageUsers'), validate(userValidation.createUser), userController.createUser)
-  .get(ideaController.getIdeas);
+    .route('/')
+    // .post(auth('manageUsers'), validate(userValidation.createUser), userController.createUser)
+    .get(ideaController.getIdeaPosts);
```
Comment (Shivz3232) [inline src/routes/v1/idea.route.js]: Can you please use the existing naming convention here?

File: src/routes/v1/idea.route.js [hunk 2]
```diff
@@ -2,14 +2,14 @@ const express = require('express');
 // const auth = require('../../middlewares/auth');
 // const validate = require('../../middlewares/validate');
 // const userValidation = require('../../validations/user.validation');
-const ideaController = require('../../controllers/idea.controller');
+const ideaController = require('../../controllers/ideas.controller/index');
```
Comment (Shivz3232) [inline src/routes/v1/idea.route.js]: You can omit the `/index`

**Comment by Shivz3232** [comment]
Hey, I have added 2 comments on the idea router. Can you please resolve these on the idea router as well as other routers (post & thoughts)?

**Review by Shivz3232 — CHANGES_REQUESTED** [review]

**Comment by khushilms** [comment]
Hi shivu, i have added the controllers for add connection, reject connection and accept connection. Can you please check if it is right?

**Comment by Shivz3232** [comment]
Closing PR, to open a new one from the newly created branch.

**Pull request closed without merging**